code/game/machinery/kitchen/microwave.dm

```diff
@@ -417,7 +438,7 @@
 			cooked.forceMove(src.loc)
 		return
 
-/obj/machinery/microwave/proc/wzhzhzh(var/seconds as num)
+/obj/machinery/microwave/proc/microwave_run(var/seconds as num) // was called wzhzhzh, for some fucking reason
```

Comment:
how dare you

Reply:
i stand by what i have done

Reply:
wow so brave

Reply:
why not simply `run` instead of `microwave_run` ?

Reply:
i could change it to run i guess, i just felt like microwave_run was a little more descriptive

Reply:
pretty sure run is a reserved keyword

Reply:
microwave run it is then

Reply:
What about `start()`? You're at /obj/machinery/microwave, there's no need to clarify it's a microwave proc. Also if you go microwave_run then it should also be microwave_empty, microwave_broken, etc for consistency with other procs

Reply:
I can do that sure